src/aws/lambda-update-state/index.spec.js

```diff
@@ -17,12 +21,25 @@ describe('lambda', () => {
           ObjectKey: 'some object key',
         },
         status: 'COMPLETE',
+        outputGroupDetails: [
+          {
+            type: 'FILE_GROUP',
+            outputDetails: [
+              {
+                outputFilePaths: [
+                  's3://dev-manu-marsha-destination/70795cbb-85b0-4742-bc89-d872fbf9fb5f/mp4/1551969052_144.mp4',
```

Comment:
Not sure why this is a lis but there might be cases in which there are several videos in this list?

Reply:
For now I don't see case where we will have several videos.

Reply:
Done